api/python/cell_census/tests/README.md

```diff
@@ -0,0 +1,47 @@
+# Test README
+
+This directory contains tests of the cell-census package API, _and_ the use of the API on the
+live "corpus", i.e., data in the public cell census S3 bucket. The tests use Pytest, and have
+Pytest marks to control which tests are run.
+
+Tests can be run in the usual manner. First, ensure you have cell-census installed, e.g., from the top-level repo directory:
+
+> pip install -e ./api/python/cell_census/
+
+Then run the tests:
+
+> pytest ./api/python/cell_census/
+
+## Pytest Marks
+
+There are two Pytest marks you can use from the command line:
+
+- live_corpus: tests that directly access the `latest` version of the Cell Census. Enabled by default.
+- expensive: tests that are expensive (ie., cpu, memory, time). Disabled by default - enable with `--expensive`. Some of these tests are _very_ expensive, ie., require a very large memory host to succeed.
+
+By default, only relatively cheap & fast tests are run. To enable `expensive` tests:
+
+> pytest --expensive ...
+
+To disable `live_corpus` tests:
+
+> pytest -m 'not live_corpus'
+
+You can also combine them, e.g.,
+
+> pytest -m 'not live_corpus' --expensive
+
+# Acceptance (expensive) tests
+
+These tests are periodically run, and are not part of CI due to their overhead.
+
+Please record:
+
+- date
+- host / instance type
+- Python & package versions and OS (tip: use tiledbsoma.show_package_versions())
+- full output of: `pytest --durations=0 --expensive ./api/python/cell_census/tests/`
```

Comment:
also, the cell census release, unless that shows up in the output?  motivation is to know what data size tests last passed for. Maybe each test could output relevant sizes on full reads, like obs.  Or output len(obs), len(var), X.nnz once.

Reply:
yes on recording the version aliased to `latest`

I'm not sure we want the tests generating a bunch of output unless it is regularly used.  I think your census version is the ideal solution.